__mocks__/console.js

```diff
@@ -0,0 +1,20 @@
+function format(entry) {
```

Comment:
Just curious.. What happens if you don't mock `console`?

Reply:
Oh is it because of the `mock-fs`? I wonder if there is a way to just isolate these lines to test: https://github.com/Expensify/Expensify.cash/pull/1587/files#diff-1eb882ec8d1b90e8df9c44344e71cf2c16545c47e0ffc3c877d8a899cf8dbd9eR61-R62

Maybe splitting them out into their own function would allow us not to have to mock fs and console, and still have the same value?

Reply:
```
 ENOENT: no such file or directory, lstat '/Users/roryabraham/ReactNativeChat/node_modules/jest-runner/node_modules/jest-message-util'

      at _jestMessageUtil (node_modules/jest-runner/node_modules/@jest/console/build/getConsoleOutput.js:19:16)

```

Reply:
I also included a link to the related GH issue where this mock is used :)